### ISSUE TITLE
dcs.action.DoScript error

### ISSUE THREAD

**Issue by jeremymoran** [pydcs/dcs#179]
Hi,

Trying to execute a simple MIST script to respawn aircraft. I have successfully loaded the MIST.lua at mission start. However when I want to execute using the `DoScript` function, in DCS I get a "DictKey_Translation_7" in the text box.

Python script:
```python
def respawn_trigger(m,groupName):
    lua_script =  "if not Group.getByName('"+groupName+"') then \
                       mist.respawnGroup('"+groupName+"', true) \
                     end"
    trig_respawn = dcs.triggers.TriggerContinious(dcs.triggers.Event.NoEvent, comment='respawn '+groupName)
    trig_respawn.add_action(dcs.action.DoScript(m.string(lua_script)))
    m.triggerrules.triggers.append(trig_respawn)
```

The end result in DCS is a "DictKey_Translation_7" as the text for the DoScript action.

Also, for more info. Here is my dictionary file from the .miz. I am only calling the respawn_trigger function once, however there are two DictKeys in the file?
```lua
dictionary=
{
	["DictKey_Translation_1"]="",
	["DictKey_Translation_2"]="",
	["DictKey_Translation_3"]="",
	["DictKey_Translation_4"]="",
	["DictKey_Translation_6"]="Excellent job!\
Intelligence reports the convoy was destroyed!\
RTB",
	["DictKey_Translation_7"]="if not Group.getByName('USSR CAP 1') then\
   mist.respawnGroup('USSR CAP 1', true)\
end",
	["DictKey_Translation_8"]="if not Group.getByName('USSR CAP 1') then\
   mist.respawnGroup('USSR CAP 1', true)\
end"
}
```

**Comment by jeremymoran** [comment]
Okay, discovered the error. Don't use the `m.string` function. String needs to be a literal string for it to work correctly.

```python
def respawn_trigger(m,groupName):
    lua_script =  f"if not Group.getByName('"+groupName+"') then\n   mist.respawnGroup('"+groupName+"', true)\nend"
    trig_respawn = dcs.triggers.TriggerContinious(comment='respawn '+groupName)
    trig_respawn.actions.append(dcs.action.DoScript(String(lua_script)))
    m.triggerrules.triggers.append(trig_respawn)
```

**Comment by DanAlbert** [comment]
I think the type annotation (and therefore the default constructor) are probably wrong and this is supposed to be `str` and not `String`? Most of pydcs doesn't need (or often support) the translation key `String` type any more.

https://github.com/pydcs/dcs/blob/2b4eb8bf99d225b37305735773d6504ad844d144/dcs/action.py#L385

You changing from `m.string` to `String()` is doing effectively that. The first argument to the `String` constructor is an ID, not the content.

**Comment by jeremymoran** [comment]
> 
> 
> I think the type annotation (and therefore the default constructor) are probably wrong and this is supposed to be `str` and not `String`? Most of pydcs doesn't need (or often support) the translation key `String` type any more.
> 
> https://github.com/pydcs/dcs/blob/2b4eb8bf99d225b37305735773d6504ad844d144/dcs/action.py#L385
> 
> You changing from `m.string` to `String()` is doing effectively that. The first argument to the `String` constructor is an ID, not the content.

Yes that was part of the issue as well. I tried String() earlier and it still didn't work.

**Comment by DanAlbert** [comment]
Worth reopening to get the default fixed IMO. I don't have those powers :)
